packages/components/Menu/src/MenuGroup/MenuGroup.tsx

```diff
@@ -39,7 +44,21 @@ export const MenuGroup = compose<MenuGroupType>({
         }
         return child;
       });
-      return <Slots.root {...mergedProps}>{childrenWithSet}</Slots.root>;
+
+      return (
+        <Slots.root {...mergedProps}>
+          <Slots.focusZone
```

Comment:
I know the slot is a Fragment on non win32 platforms, but it still feels a bit weird to see the name here. I don't know I have a better idea, so I guess not blocking